### DISCUSSION TITLE
XR support ?

### DISCUSSION THREAD

**Discussion by fromEON** [o3de/o3de#10737] [Ideas]
Is there any future plans targetting platforms for VR like Google Cardboard, Oculus Quest, htc Vive, windows mixed reality platforms?

Also coming to AR there is already support for Android and iOS. Can I expect ARCore and ARkit for respective platforms

**Comment by lmbr-pip** [comment]
VR support is being worked on, but am unsure of state at the moment, as its not my area of expertise. 

The RFC to initiate AR/VR work was approved in April: https://github.com/o3de/sig-graphics-audio/issues/39

Work has started, and some of it has made it to the development branch. For example, https://github.com/o3de/o3de/pull/10170 and https://github.com/o3de/o3de/pull/9664 

If you have questions or would like more info, I recommend asking in the sig-graphics-audio chat 😄 on Discord

**Comment by moraaar** [comment]
An initial integration of OpenXR with support for Meta Quest 2 was completed in November 2022.

You can check the roadmap initiative for more information: https://github.com/o3de/o3de/issues/12372

**Reply by hultonha** [reply]
These links might be useful for reference too

https://github.com/o3de/o3de-extras/tree/development/Gems/OpenXRVk
https://github.com/o3de/o3de-extras/tree/development/Gems/XR
https://github.com/o3de/o3de-extras/tree/development/Projects/OpenXRTest

The o3de-extras repo is definitely a good place to start 🙂 Support is currently experimental at the moment as far as I know though.
